myproject/supermarket_manager/views.py

```diff
@@ -161,24 +168,26 @@ def add_permission(request):
         - JsonResponse: A JSON response indicating the result of the add operation.
     """
 
-    if request.method == "POST":
-        data = json.loads(request.body)
-        permission_name = data.get("permission_name", None)
-        description = data.get("description", None)
-
-        if permission_name is not None and description is not None:
-            try:
-                permission = Permission(
-                    permission_name=permission_name, description=description
-                )
-                permission.save()
-                return JsonResponse({"message": ADDED}, status=201)
-            except IntegrityError:
-                return JsonResponse({"message": EXISTS}, status=400)
-        else:
-            return JsonResponse({"message": REQUIRED}, status=400)
+    if request.method != "POST":
+        return JsonResponse({"message": INVALID_METHOD}, status=405)
+    
+    data = json.loads(request.body)
+    permission_name = data.get("permission_name", None)
+    description = data.get("description", None)
 
-    return JsonResponse({"message": INVALID_METHOD}, status=405)
+    if permission_name is not None and description is not None:
```

Comment:
Can we use the or operator?

Reply:
Done

Reply:
@hothanhtuan1209, The way you did it is not correct, please try another way.